x-pack/plugins/canvas/common/lib/fonts.js

```diff
@@ -4,53 +4,102 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 export const americanTypewriter = {
-  label: 'American Typewriter',
+  label: i18n.translate('xpack.canvas.lib.fonts.americanTypewriterFontLabel', {
```

Comment:
Please check it works for other locales. We have some issue for translating constants. If it does not work please use functions instead

Reply:
We shouldn't be translating font names.